### ISSUE TITLE
Alert dialog boxes on Windows should have shortcut keys on buttons

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#6400]
Create a new file in RStudio for Windows Desktop, make some changes, and don't save it. Then try to close it and you'll get this:

![screenshot of RStudio's Save / Don't Save dialog box](https://user-images.githubusercontent.com/10569626/75914750-2b751a00-5e0a-11ea-9d77-58a71881731e.png)

Notice there are no underlined letters in the Save and Don't Save buttons.

Try the same thing with notepad and the "S" in Save and the "n" in Don't Save are underlined. You can thus trigger Save via the S key, and Don't Save via the N key. This is a common behavior on Windows desktop applications and very helpful for a keyboard user.

![screenshot of notepad's Save / Don't Save dialog box](https://user-images.githubusercontent.com/10569626/75915424-50b65800-5e0b-11ea-82fa-0dcfbdf5a3e0.png)
 
Another common case is "Yes No" alerts. Should be able to pick Yes via the letter Y, No via the letter N.

**Comment by balthasars** [comment]
I'd appreciate it if this could be reconsidered (although now closed) 👍🏼

**Comment by ronblum** [comment]
Closing because this is no longer present in some system dialogs, and we rely on the consistency.
